Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,16 +1,58 @@
-function setAlarm() {}
+var input_time = document.querySelector("#alarmSet");
+var title_time = document.querySelector("#titleTime");
+var timeRemaining = document.querySelector("#timeRemaining");
+var bUserStopped = false;
+function setAlarm() {
+  
+  let totalSeconds = parseInt(input_time.value, 10);
+  if (isNaN(totalSeconds) || totalSeconds <= 0) {
+    alert("Please enter a valid number of seconds.");
+    return;
+  }
+  
+  const title_second = input_time.value % 60;
+  const title_minute = (input_time.value - title_second) / 60;
+
+  title_time.innerText = `${title_minute.toString().padStart(2, "0")}:${title_second.toString().padStart(2, "0")}`;
+
+  let timer = setInterval(() =>{
+    let seconds = totalSeconds % 60;
+    let minutes = (totalSeconds - seconds) / 60;
+    timeRemaining.innerText = `Time Remaining: ${minutes.toString().padStart(2, "0")}:${seconds.toString().padStart(2, "0")}`;
```

Comment:
You've written the same code (to take a number and convert it to a padded string) twice here, and two more times above for the title.

Can you think how you could avoid writing that same code four times?